beetsplug/info.py

```diff
@@ -52,7 +53,11 @@ def emitter():
             tags[field] = getattr(mf, field)
         tags['art'] = mf.art is not None
         tags['path'] = displayable_path(path)
+        # create a temporary Item to take advantage of __format__
+        tags['item'] = Item(db=None, **tags)
```

Comment:
I like that idea, although after playing a bit with potential solutions returning `Item`s directly might have subtle implications on the output. Currently:
* if the `info` command is passed a file, it only prints the fields present on the `MediaFile` (plus a boolean `art` field, and a `genres` field).
* if the `info` command is passed a library item, it prints all the fields, including the empty ones.
```
$ ./beet info test/rsrc/image.mp3 
/home/x/venvs/beets/git/beets/test/rsrc/image.mp3
       art: True
  bitdepth: 0
   bitrate: 68536
  channels: 1
    format: MP3
    genres: 
    length: 1.0
samplerate: 44100

$ ./beet info -l foo
/tmp/music/artist/album/track.mp3
acoustid_fingerprint: 
         acoustid_id: 
               added: 2015-10-15 19:04:53
               album: Album
            album_id: 5
...
               track: 1
          tracktotal: 12
                year: 1988
```
This seems to be mainly due to the dict being built from the `MediaFile` directly, and as a result, having `None` as a value for the tags not present on the file (`print_data` has a `if value is not None:` line which explicitly checks for `None`).

My main concern is that it might be hard to find out what fields are really to be printed due to being on the `MediaFile` if the emitters return just `Items` directly (for example, fields on the mediafile stored as a zero or empty string, or fields initialized by the `Item` constructor with a default empty value, etc), where the difference between a non-present field and an "empty" one is not that clear.

Perhaps it would make sense to have `tag_data_emitter` produce `MediaFile`s, and `library_data_emitter` produce `Items`, having the flattening handle the differences?